backend/services/soft_deletion.py

```
@@ -48,27 +48,24 @@ def soft_delete_scenario(
             
-            print(f"[DEBUG] Successfully deleted scenario '{scenario_title}' (ID: {scenario_id})")
             return True
             
         except Exception as e:
             self.db.rollback()
-            print(f"Error soft deleting scenario {scenario_id}: {e}")
             return False
     
-    def _archive_user_progress_for_scenario(self, scenario_id: int, reason: str):
-        """Archive user progress for a deleted scenario"""
+    def _soft_delete_user_progress_for_scenario(self, scenario_id: int, reason: str):
```

Ruff (0.14.1)

Missing return type annotation for private function `_soft_delete_user_progress_for_scenario`

Add return type annotation: `None`

(ANN202)